src/Compilers/CSharp/Test/Semantic/Semantics/QueryTests.cs

```diff
@@ -3597,7 +3639,9 @@ where x.ToString() == y.ToString()
                     Children(1):
                         IOperation:  (OperationKind.None, Type: null, IsInvalid) (Syntax: 'x.ToString')
                           Children(1):
-                              IOperation:  (OperationKind.None, Type: null, IsInvalid) (Syntax: 'x')
+                              IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid, IsImplicit) (Syntax: 'x')
```

Comment:
>IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid, IsImplicit) (Syntax: 'x') [](start = 30, length = 87)

It feels like there should be an explicit node associated with (Syntax: 'x') and probably it should be this node.

Reply:
This seems to be due to incorrect implementation of IsImplicit for bad expression and statements in both C# and VB operation factories. Filed https://github.com/dotnet/roslyn/issues/23164 to track this issue.